konfera/settings.py

```diff
@@ -29,3 +29,6 @@
 """
 REGISTER_EMAIL_NOTIFY = getattr(settings, 'REGISTER_EMAIL_NOTIFY', False)
 REGISTER_EMAIL_BCC = getattr(settings, 'REGISTER_EMAIL_BCC', [])
+
+"""Notify after submitting proposal"""
+PROPOSAL_NOTIFY = getattr(settings, 'PROPOSAL_NOTIFY', False)
```

Comment:
Lets use generic variable naming, I suggest `PROPOSAL_EMAIL_NOTIFY` so it matches `REGISTER_EMAIL_NOTIFY`, eg. all other notification should be `SOMETHING_EMAIL_NOTIFY`